File: BE.Core.FW/Backend/Business/ExamCalendar/ExamCalendarModel.cs
using System.ComponentModel.DataAnnotations;

namespace Backend.Business.ExamCalendar
{
    public class ExamCalendarModel
    {
        public Guid Id { get; set; }
        public Guid HeaderQuarterId { get; set; }
        public Guid Room { get; set; }
        public DateTime DateTest { get; set; }
        public DateTime EndDateRegister { get; set; }
        public Guid ExamShift { get; set; }
        public string TimeTest { get; set; } = string.Empty;
        public string Note { get; set; } = string.Empty;
        public string ExamId { get; set; } = string.Empty;
        public int Status { get; set; }
        public int QuantityCandidate { get; set; }
        public int Registed { get; set; }
        public int Limit { get; set; }
    }
}

File: FE.Core.FW/src/apis/models/ExamCalendarModel.ts
/* istanbul ignore file */
/* tslint:disable */
/* eslint-disable */

export type ExamCalendarModel = {
  id?: string;
  headerQuarterId?: string;
  room?: string;
  dateTest?: string;
  endDateRegister?: string;
  examShift?: string;
  timeTest?: string | null;
  note?: string | null;
  examId?: string | null;
  status?: number;
  quantityCandidate?: number;
  registed?: number;
  limit?: number;
};

